src/Session.cc

```diff
@@ -1571,6 +1571,12 @@ void Session::SendEvent(CARTA::EventType event_type, uint32_t event_id, const go
         std::pair<std::vector<char>, bool> msg;
         if (_connected) {
             while (_out_msgs.try_pop(msg)) {
+                auto expected_buffered_amount = msg.first.size() + _socket->getBufferedAmount();
+                if (expected_buffered_amount > MAX_BACKPRESSURE) {
+                    spdlog::warn(
+                        "Client {} [{}] WebSocket buffered amount {} (bytes) exceeds the max backpressure! May losing some messages..",
```

Comment:
Change "May losing" to "May lose"; remove extra trailing full stop.

Reply:
Also change "Client {} [{}] WebSocket buffered amount {} (bytes) exceeds the max backpressure!" to "Exceeded maximum backpressure: client {} [{}]. Buffered amount: {} (bytes)."